web-console/4.0-designs/code-editors/code-editor.md

```diff
@@ -0,0 +1,46 @@
+# Code Editor
+
+## Edit YAML
+
+![pic 1](img/1.png)
+
+- The code area is borderless at the top and has the button bar at the bottom as a separator.
+- The Edit action can be removed from the Actions dropdown on every detail page. Users can access the tab as needed.
```

Comment:
The "Edit" inside the action dropdown is because _list view and details view share the same action dropdown_. This enables users to directly enter YAML for editing from the list view. 
Not that we can't remove the "Edit" from the action dropdown on the Details view, but keeps the "Edit" inside the action dropdown on the List view. Just brings up this to see if we really need to maintain two versions of action dropdown (_List View version_ vs _Details View version_).